stable/awsebscsiprovisioner/Chart.yaml

```diff
@@ -1,12 +1,12 @@
 apiVersion: v1
-appVersion: "0.4.0"
+appVersion: "0.5.0"
```

Comment:
Im seeing this in the test log https://teamcity.mesosphere.io/viewLog.html?buildId=2646542&buildTypeId=ClosedSource_Konvoy_ChartPr&tab=buildLog&_focus=847#_state=205

Reply:
uh thanks @dkoshkin will recheck

Reply:
Is this still relevant?

Reply:
solved e2e ;) so not relevant anymore :)